src/components/Menu/index.tsx

```diff
@@ -115,14 +120,9 @@ export function Item({children, label, style, onPress, ...rest}: ItemProps) {
       accessibilityHint=""
       accessibilityLabel={label}
       onPress={e => {
+        control?.close()
         onPress(e)
-
-        if (!e.defaultPrevented) {
-          control?.close()
-        }
```

Comment:
what's wrong with this?

Reply:
[`3fabcc2` (#5581)](https://github.com/bluesky-social/social-app/pull/5581/commits/3fabcc20d8e157431c3fe30aaf1a84af5d03a385)

Reverted in this PR, not picking because types are still fucked in this PR (didn't normalize the pressables yet)

Reply:
I don't see any usages of this, but I may have missed one. We have the same functionality in `Link` so that you can exit the action on failure e.g. show a toast or something. Seems harmless to keep this?

Reply:
Yea I added it back in the commit above. Just wasn't typed correctly.

Reply:
Oh I hadn't refreshed the page, was replying to your prev 👍